files/register-server.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -eo pipefail
```

Comment:
Please make sure that this is backported to https://github.com/appuio/terraform-openshift4-exoscale